src/sentry/static/sentry/app/components/selectInput.jsx

```diff
@@ -2,6 +2,15 @@ import React from 'react';
 import jQuery from 'jquery';
 
 const SelectInput = React.createClass({
+  propTypes: {
+    disabled: React.PropTypes.bool,
+    multiple: React.PropTypes.bool,
+    required: React.PropTypes.bool,
+    placeholder: React.PropTypes.string,
+    value: React.PropTypes.string,
+    onChange: React.PropTypes.func,
```

Comment:
> Seems that you generally don't have one.

I'd prefer not to have trailing comma. It's not common practice in JavaScript like it is in Python.